### ISSUE TITLE
Properties not found when using Grouped UITableView

### ISSUE THREAD

**Issue by bruno01989** [schibsted/layout#65]
#### Summary

When adding a `UITableView` to a Layout .xml file and setting its `style` property to `grouped`, the properties can no longer be found by Layout.

#### How to reproduce?

- Add `UITableView` in the .xml file and set its `style` property to `grouped`
- Add a `UIView` to be used as a header inside the `UITableView` declaration
- Add a `UITableViewCell` and add the following property: `textLabel.text="{title}"`

#### What happens?

For as long as the `style` property of the `UITableView` is set to `grouped`, Layout won't be able to find the `title` property that is used inside the `UITableViewCell`

Removing the `UIView` that is used as a header also fixes the issue.

Some sample code:

```xml

<UIView>

    <UITableView
        outlet="tableView"
        style="grouped">

        <UIView
            height="40"/>

        <UITableViewCell
            reuseIdentifier="tableViewCell"
            textLabel.text="{title}"/>

    </UITableView>

</UIView>


```

```Swift
    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let node = tableView.dequeueReusableCellNode(withIdentifier: "tableViewCell")
        node?.setState(["title": "This is a title"])
        guard let cell = node?.view as? UITableViewCell else { return UITableViewCell() }
        return cell
    }
```

Gives the following error:

![simulator screen shot - iphone 5s - 2017-10-20 at 09 33 29](https://user-images.githubusercontent.com/12001171/31809890-ef139fea-b579-11e7-80a0-b0ba48370530.png)

**Comment by nicklockwood** [comment]
Thanks, I’ll look into it.

**Comment by nicklockwood** [comment]
Fixed in 0.5.5

**Comment by sergeymild** [comment]
Thanks
